js/client-teambuilder.js

```diff
@@ -1699,19 +1699,20 @@
 
 			if (template.speciesid === 'meowstic') {
 				smogdexid = 'meowstic-m';
-			} else if (smogdexid === 'rotom' || smogdexid === 'deoxys' || smogdexid === 'kyurem' || smogdexid === 'giratina' || smogdexid === 'shaymin' || smogdexid === 'tornadus' || smogdexid === 'thundurus' || smogdexid === 'landorus' || smogdexid === 'pumpkaboo' || smogdexid === 'gourgeist' || smogdexid === 'arceus' || smogdexid === 'meowstic' || smogdexid === 'hoopa') {
-				if (template.forme) smogdexid += '-' + toId(template.forme);
+			} else if (template.forme) {
+				if (template.speciesid === 'vivillon-pokeball' || template.speciesid === 'vivillon-fancy') smogdexid = 'vivillon';
```

Comment:
I only included the vivillon formes because keldeo-r redirects to keldeo, pikaformes have their own pages, and basculin blue stripe redirects as well (same with ash greninja). the vivillon formes neither have their own pages nor do they redirect to vivillon, hence why i only included them